### PR TITLE
(style) profile view

### PR DESCRIPTION
**Pull request by lschwall**

### CODE REVIEW

File: server/routes/customer.js [hunk 1]
```diff
@@ -82,12 +82,13 @@ customerRouter.post('/register', async (req, res) => {
     .then((customers) => {
       if (customers.length > 0) {
         res.send('customer');
+        console.info('CUSTOMER FOUND', customers[0]);
```
Comment (jstrzesz) [inline server/routes/customer.js:85]: remove

File: server/routes/customer.js [hunk 2]
```diff
@@ -123,7 +123,6 @@ customerRouter.post('/create', (req, res) => {
           })
           .catch((err) => {
```
Comment (jstrzesz) [inline server/routes/customer.js]: your .catch is in the wrong spot. it should be one level up to match with the first .then

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: server/routes/customer.js (hunk 1 above)
Comment (lschwall) [reply server/routes/customer.js:85]: This will be removed on final sweep tonight after styling

**Review by tenhold — APPROVED** [review]

**Review by christophermbooth — APPROVED** [review]

**Pull request merged**